File: calculapreco.java
public class Produto {
    String nome;
    double preco;
    double desconto;

    public Produto(String nome, double preco, double desconto) {
        this.nome = nome;
        this.preco = preco;
        this.desconto = desconto;
    }

    public double calcularPrecoFinal() {
        return preco - (preco * desconto / 100);
    }

    public static void main(String[] args) {
        // Exemplo de uso com desconto de 25% (alterado de 15 para 25)
        Produto produto = new Produto("Fone de ouvido", 200, 25);
        
        // INCLUSÃO: Imprime o preço inicial
        // (Cópia e edição da linha de impressão do preço final)
        System.out.printf("Preço inicial de %s: R$ %.2f%n", produto.nome, produto.preco);

        // Imprime o preço final (comando original)
        System.out.printf("Preço final de %s: R$ %.2f%n", produto.nome, produto.calcularPrecoFinal());
    }
}

File: calculapreco.cs
using System;
class Produto
{
    public string Nome { get; set; }
    public double Preco { get; set; }
    public double Desconto { get; set; }

    public double CalcularPrecoFinal()
    {
        return Preco - (Preco * Desconto / 100);
    }
}
class Program
{
    static void Main()
    {
        Produto produto = new Produto
        {
            Nome = "Fone de ouvido",
            Preco = 200,
            // Valor do Desconto alterado de 15 para 25
            Desconto = 25 
        };
        
        // INCLUSÃO: Imprime o preço inicial (Preco)
        // Linha copiada e editada
        Console.WriteLine($"Preço inicial de {produto.Nome}: R$ {produto.Preco:F2}");

        // Imprime o preço final (CalcularPrecoFinal())
        Console.WriteLine($"Preço final de {produto.Nome}: R$ {produto.CalcularPrecoFinal():F2}");
    }
}
